cdap-app-fabric-tests/src/test/java/co/cask/cdap/WorkflowAppWithScopedParameters.java

```diff
@@ -179,25 +207,30 @@ public void destroy() {
       Preconditions.checkArgument(ONE_MR.equals(nodeState.getNodeId()));
       Preconditions.checkArgument(nodeState.getRunId() != null);
       Preconditions.checkArgument(NodeStatus.COMPLETED == nodeState.getNodeStatus());
+      Preconditions.checkArgument(OneMR.getFieldLineageOperations() == nodeState.getFieldLineageOperations());
```

Comment:
should this be equals()

Reply:
yes.. thats correct. Actually its written in `destroy` so throwing any exception does not fail the test case. Updated test case so that we add some data in the WorkflowToken towards the end of the destroy and checking for that in the test case now.